src/style.css

```diff
@@ -1 +1,75 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  margin-top: 0;
+  padding-top: 20px;
+}
+
+.search__input {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  outline: none;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.search__input::placeholder {
+  color: inherit;
+}
+
+.search__input--large {
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  background-size: 19px;
+  background-position: 26px center;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__input--small {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px center;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search__input--large:hover,
+.search__input--small:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
```

Comment:
If both modifiers apply the same style you should consider moving the style to the main block!